lib/anidb/media_category_assoc_importer.rb

```diff
@@ -0,0 +1,94 @@
+class AnidbAssocMediaCategoryImport
```

Comment:
A namespace might be good? 🤔  Maybe `AnidbCategoryImport` (`lib/anidb_category_import/media_import.rb`, etc.)?